
# HG changeset patch
# User Vekhir
# Date 1704916469 -3600
# Node ID 2c92499daa677c414e1de5782539a4847fa9c5c1
# Parent  cf580d9ff7ef6bdebd8597b17a1c3788b11f11a8
Fix server build with modern mtl library

diff -r cf580d9ff7ef -r 2c92499daa67 gameServer/ClientIO.hs
--- a/gameServer/ClientIO.hs	Fri Oct 06 22:37:44 2023 +0200
+++ b/gameServer/ClientIO.hs	Wed Jan 10 20:54:29 2024 +0100
@@ -20,6 +20,7 @@
 module ClientIO where
 
 import qualified Control.Exception as Exception
+import Control.Monad
 import Control.Monad.State
 import Control.Concurrent.Chan
 import Control.Concurrent
diff -r cf580d9ff7ef -r 2c92499daa67 gameServer/HWProtoChecker.hs
--- a/gameServer/HWProtoChecker.hs	Fri Oct 06 22:37:44 2023 +0200
+++ b/gameServer/HWProtoChecker.hs	Wed Jan 10 20:54:29 2024 +0100
@@ -20,6 +20,7 @@
 module HWProtoChecker where
 
 import Data.Maybe
+import Control.Monad
 import Control.Monad.Reader
 --------------------------------------
 import CoreTypes
diff -r cf580d9ff7ef -r 2c92499daa67 gameServer/HWProtoCore.hs
--- a/gameServer/HWProtoCore.hs	Fri Oct 06 22:37:44 2023 +0200
+++ b/gameServer/HWProtoCore.hs	Wed Jan 10 20:54:29 2024 +0100
@@ -19,6 +19,7 @@
 {-# LANGUAGE OverloadedStrings #-}
 module HWProtoCore where
 
+import Control.Monad
 import Control.Monad.Reader
 import Data.Maybe
 import qualified Data.ByteString.Char8 as B
diff -r cf580d9ff7ef -r 2c92499daa67 gameServer/HWProtoLobbyState.hs
--- a/gameServer/HWProtoLobbyState.hs	Fri Oct 06 22:37:44 2023 +0200
+++ b/gameServer/HWProtoLobbyState.hs	Wed Jan 10 20:54:29 2024 +0100
@@ -21,6 +21,7 @@
 
 import Data.Maybe
 import Data.List
+import Control.Monad
 import Control.Monad.Reader
 import qualified Data.ByteString.Char8 as B
 --------------------------------------
diff -r cf580d9ff7ef -r 2c92499daa67 gameServer/HandlerUtils.hs
--- a/gameServer/HandlerUtils.hs	Fri Oct 06 22:37:44 2023 +0200
+++ b/gameServer/HandlerUtils.hs	Wed Jan 10 20:54:29 2024 +0100
@@ -18,6 +18,7 @@
 
 module HandlerUtils where
 
+import Control.Monad
 import Control.Monad.Reader
 import qualified Data.ByteString.Char8 as B
 import Data.List
diff -r cf580d9ff7ef -r 2c92499daa67 gameServer/ServerState.hs
--- a/gameServer/ServerState.hs	Fri Oct 06 22:37:44 2023 +0200
+++ b/gameServer/ServerState.hs	Wed Jan 10 20:54:29 2024 +0100
@@ -30,6 +30,7 @@
     io
     ) where
 
+import Control.Monad
 import Control.Monad.State.Strict
 import Data.Set as Set(Set)
 import Data.Word
diff -r cf580d9ff7ef -r 2c92499daa67 gameServer/Votes.hs
--- a/gameServer/Votes.hs	Fri Oct 06 22:37:44 2023 +0200
+++ b/gameServer/Votes.hs	Wed Jan 10 20:54:29 2024 +0100
@@ -19,6 +19,7 @@
 {-# LANGUAGE OverloadedStrings #-}
 module Votes where
 
+import Control.Monad
 import Control.Monad.Reader
 import Control.Monad.State.Strict
 import ServerState
